docs/workflows/common-workflow-steps/fod-mobile-assesment.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Fortify on Demand Mobile Assessment
+description: Optimize mobile app security with Fortify on Demand Mobile Assessment. Comprehensive testing for robust, secure mobile applications.
+tags: [mobile, mobile ci/cd, security, integration, ios, android]
+sidebar_position: 4
+---
+
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Fortify on Demand Mobile Assessment
+
+[Fortify on Demand Mobile Assessment](https://www.opentext.com/products/fortify-on-demand) provides a robust solution for securing mobile applications by identifying vulnerabilities before they can be exploited. This comprehensive cloud-based service streamlines the security testing process, ensuring your mobile apps meet the highest standards of security.
+
+You can seamlessly integrate Fortify on Demand Mobile Assessment into your workflow with Appcircle, facilitating easy setup and utilization within your existing development processes.
+
+### Prerequisites
```

Comment:
Flutter iOS and Android should be added too. 
The Prerequisites table should be under each platform heading.
You can examine it as an example:
- https://docs.appcircle.io/workflows/common-workflow-steps/firebase-app-distribution#prerequisites